1.8/usage/registry.md

```diff
@@ -0,0 +1,167 @@
+---
+post_title: Deploying a private Docker registry using VIPs
+nav_title: Private Docker Registry
+menu_order: 40
+---
+
+You can use a number of (hosted) [options](https://mesosphere.com/blog/2015/10/14/docker-registries-the-good-the-bad-the-ugly/) to store and manage your Docker images, but you can also set up a private Docker registry yourself and run it within your DC/OS cluster. This private registry is based on the [Docker registry image](https://docs.docker.com/registry/) and stores images in a system using one of the available storage drivers.
+
+The advantages of a private Docker registry are:
+
+- They can be securely deployed in your local environment (DC/OS cluster).
+- Reduced latency for pushing and pulling images, leading to faster deployments.
+
+This article describes how to set up a private Docker registry in a DC/OS cluster and make it accessible through a [VIP](docs/1.8/usage/service-discovery/load-balancing-vips/virtual-ip-addresses/). 
+
+## Security options
+
+Before you set up a private registry, you should decide how you want to set up security. There are three basic options for configuring security in a private registry. 
+
+1. __Trusted Certificate Authority__ This is the recommended option for production workloads. This option requires requesting a certificate from a trusted CA and existing naming configurations to resolve the registry. As an added benefit, you do not have to configure the Docker daemons to trust the certificate.
+1. __Self-signed Certificate__ This option provides decent security and is more manageable.
+1. __Insecure__ This option should be avoided when possible. It may even conflict with Docker-in-Docker containers such as Jenkins.
+
+For more information on setting up a Docker Trusted registry see [deploying a registry server](https://docs.docker.com/registry/deploying/) via the Docker docs.
+
+## Naming and discovery options
+
+After the registry is deployed clients will need to access it. When you create the certificate, you should consider the naming and discovery approach used to access the service.  You can use any of the [service discovery](/docs/1.8/usage/service-discovery/) options available in DC/OS:
+
+- __Mesos-DNS__ with host port mapping is an easy option, but requires that ports are managed as resources in the cluster.
+- __Minuteman__ provides a stable endpoint and you can use either the VIP or a DNS entry that references the VIP. A DNS entry can add a bit more complexity but improves consumption of the service.
```

Comment:
Change `Minuteman` to `Virtual IP`. VIP is the official feature name, not minuteman.